tests/system/test_client.py

```diff
@@ -1804,6 +1799,11 @@ def test_dbapi_fetch_w_bqstorage_client_large_result_set(self):
                 ("id", 3),
                 ("timestamp", datetime.datetime(2006, 10, 9, 18, 40, 33, tzinfo=UTC)),
             ],
+            [
```

Comment:
Not clear how the changes in this file relate to the feature.  Just a side fixup?

Reply:
The expected value was changed because the public table somehow lost the first row of the table (`id=1`) around 12/31, so the system test has been failing.